scarb/src/ops/package.rs

```diff
@@ -156,6 +156,20 @@ fn list_one_impl(
 }
 
 fn prepare_archive_recipe(pkg: &Package) -> Result<ArchiveRecipe> {
+    ensure!(
+        pkg.manifest.targets.iter().any(|x| x.is_lib()),
+        formatdoc! {
```

Comment:
```suggestion
        formatdoc!(
```

we use `{` for `indoc!` et all when there is a single argument. then, rustfmt glues the brace and string literal together instead of making an extra indentation

here we indent anyway, so let's go with `(` which is better handled by rustfmt